public/index.html

```diff
@@ -26,6 +26,7 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <div id="modal"></div>
```

Comment:
It's ok this div for the modal portal but normally this can be implemented as part of a React component itself